worker/src/main/java/gov/cms/ab2d/worker/service/WorkerServiceImpl.java

```diff
@@ -17,16 +18,26 @@
     private final JobPreProcessor jobPreprocessor;
     private final JobProcessor jobProcessor;
 
+    @Value("${delay-processing}")
+    private String delayProcessing;
 
     @Override
     public void process(String jobUuid) {
+        // This exists for the e2e tests so that when a status request comes in after the export, there is enough time
+        // so that the job doesn't get processed and finishes. This way incomplete statuses can be tested. This should never
+        // run in any other environment
+        if (delayProcessing.equals("true")) {
+            try {
+                Thread.sleep(10000);
+            } catch (InterruptedException e) {
+                log.error("Thread was interrupted while sleeping for e2e tests", e);
+            }
+        }
+
```

Comment:
We should address this by increasing the number of patients per contract, so that job processing takes longer.